src/gui/transferlistsortmodel.cpp

```diff
@@ -85,56 +86,90 @@ void TransferListSortModel::disableTrackerFilter()
 
 bool TransferListSortModel::lessThan(const QModelIndex &left, const QModelIndex &right) const
 {
-    switch (sortColumn()) {
+    return lessThan_impl(left, right, sortColumn());
+}
+
+bool TransferListSortModel::lessThan_impl(QModelIndex left, QModelIndex right, int sortColumn) const
+{
+    left = left.sibling(left.row(), sortColumn);
+    right = right.sibling(right.row(), sortColumn);
+
+    const QVariant leftValue = left.data(Qt::UserRole);
+    const QVariant rightValue = right.data(Qt::UserRole);
```

Comment:
I would prefer:
```c++
const QVariant leftValue = left.sibling(left.row(), sortColumn).data(Qt::UserRole);
const QVariant rightValue = right.sibling(right.row(), sortColumn).data(Qt::UserRole);
```
And then use const references for `left` & `right` function parameters.